docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p><code>MergeInteractionsToSqrtISwap</code>  removed the entangling gate b/w the ancilla and first qubit. I think <code>MergeInteractionsToSqrtISwap</code> might not be the right choice here because the ideal circuit unitary is   just <code>Identity</code> , so if we merged all operations (not just one/two qubit), and tried to decompose we'd just get an empty circuit as the unitary would be identity ?</p><p> </p><p>The fact that we are trying to run a circuit with an identity unitary to understand device specific properties make it hard to run meaningful optimizations here because every optimization will end up changing the meaning of what we want to achieve by running the circuit? </p><p> </p><p>I'm not sure if this is a clear example for demonstration purpose. I imagine other readers could also get confused and have similar questions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Still removes the two-qubit gate between ancilla &amp; 1st qubit and may lead to some confusion as you note, but I think we settled offline on keeping this example circuit. </p>

Reply:
Sounds good.